src/test/resources/allfields/output/input-ruimtereis01/bag/metadata/files.xml

```diff
@@ -4,7 +4,7 @@
     <file filepath="data/path/to/a/random/video/hubble.mpg">
         <dcterms:type>http://schema.org/VideoObject</dcterms:type>
         <dcterms:format>video/mpeg</dcterms:format>
-        <dcterms:title>hubble.mpg</dcterms:title>
+        <dcterms:title>video about the hubble space telescoop</dcterms:title>
```

Comment:
```suggestion
        <dcterms:title>video about the hubble space telescope</dcterms:title>
```

Reply:
good catch, but given that it is in a number of places, I'll fix it manually